scripts/slurmsync.py

```diff
@@ -224,6 +225,18 @@ def find_node_status(nodename):
         if not state.base.startswith("DOWN"):
             return NodeStatus.terminated
     elif (state is None or "POWERED_DOWN" in state.flags) and inst.status == "RUNNING":
+        log.info("%s is potential orphan node", nodename)
+        age_threshold_seconds = 90
+        inst_seconds_old = _seconds_since_timestamp(inst.creationTimestamp)
+        log.info("%s state: %s, age: %0.1fs", nodename, state, inst_seconds_old)
+        if inst_seconds_old < age_threshold_seconds:
+            log.info(
+                "%s not marked as orphan, it started less than %ds ago (%0.1fs)",
```

Comment:
nit. please use f-string

Reply:
For logging specifically, I get [this warning](https://pylint.readthedocs.io/en/latest/user_guide/messages/warning/logging-fstring-interpolation.html). I realize it does not really matter since we log at info level. But I figure someone has put some thought into it.

Reply:
Ack.